src/main/java/frc/team2412/robot/commands/intake/AllInRumbleCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.team2412.robot.Controls;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+
+public class AllInRumbleCommand extends SequentialCommandGroup {
+	private final IntakeSubsystem intakeSubsystem;
+	private final Controls controls;
+
+	public AllInRumbleCommand(IntakeSubsystem intakeSubsystem, Controls controls) {
+		this.intakeSubsystem = intakeSubsystem;
+		this.controls = controls;
+
+		addCommands(new AllInCommand(intakeSubsystem, controls), new WaitCommand(0.5));
+	}
+
+	public void end() {
```

Comment:
Note that we will want to call the superclass's `end()` method to make sure the command group responds to cancellations properly.